diff --git a/smartcontract/service/neovm/config.go b/smartcontract/service/neovm/config.go
index 6c0f3ec..42513a1 100644
--- a/smartcontract/service/neovm/config.go
+++ b/smartcontract/service/neovm/config.go
@@ -49,8 +49,8 @@ var (
 
 	PER_UNIT_CODE_LEN    int = 1024
 	METHOD_LENGTH_LIMIT  int = 1024
-	DUPLICATE_STACK_SIZE int = 1024 * 2
-	VM_STEP_LIMIT        int = 400000
+	DUPLICATE_STACK_SIZE int = 1024 * 2 * 100
+	VM_STEP_LIMIT        int = 400000*100
 
 	// API Name
 	ATTRIBUTE_GETUSAGE_NAME = "Ontology.Attribute.GetUsage"
diff --git a/vm/neovm/params.go b/vm/neovm/params.go
index 158cc86..58f387a 100644
--- a/vm/neovm/params.go
+++ b/vm/neovm/params.go
@@ -20,7 +20,7 @@ package neovm
 
 const (
 	STACK_LIMIT               uint32 = 2 * 1024
-	MAX_INVOCATION_STACK_SIZE        = 1024
+	MAX_INVOCATION_STACK_SIZE        = 1024 *100
 	MAX_SIZE_FOR_BIGINTEGER          = 32
 	MAX_BYTEARRAY_SIZE        uint32 = 1024 * 1024
 	MAX_ARRAY_SIZE            uint32 = 1024
